sdk/storage/test-resources.bicep

```diff
@@ -291,30 +295,30 @@ resource fileDataContributorRoleAssignment 'Microsoft.Authorization/roleAssignme
-output STORAGE_CONNECTION_STRING string = 'DefaultEndpointsProtocol=https;AccountName=${accountName};AccountKey=${listKeys(storageAccount.id, storageApiVersion).keys[0].value};EndpointSuffix=${storageEndpointSuffix}'
+output ACCOUNT_KEY string = storageAccount.listKeys(storageApiVersion).keys[0].value
+output ACCOUNT_SAS string = '?${storageAccount.listAccountSas(storageApiVersion, accountSasProperties).accountSasToken}'
+output STORAGE_CONNECTION_STRING string = 'DefaultEndpointsProtocol=https;AccountName=${accountName};AccountKey=${storageAccount.listKeys(storageApiVersion).keys[0].value};EndpointSuffix=${storageEndpointSuffix}'
 
 output DFS_ACCOUNT_NAME string = datalakeAccountName
-output DFS_ACCOUNT_KEY string = listKeys(datalakeAccount.id, storageApiVersion).keys[0].value
-output DFS_ACCOUNT_SAS string = '?${listAccountSas(datalakeAccountNameTidy, storageApiVersion, accountSasProperties).accountSasToken}'
-output DFS_STORAGE_CONNECTION_STRING string = 'DefaultEndpointsProtocol=https;AccountName=${datalakeAccountName};AccountKey=${listKeys(datalakeAccount.id, storageApiVersion).keys[0].value};EndpointSuffix=${storageEndpointSuffix}'
+output DFS_ACCOUNT_KEY string = datalakeAccount.listKeys(storageApiVersion).keys[0].value
+output DFS_ACCOUNT_SAS string = '?${datalakeAccount.listAccountSas(storageApiVersion, accountSasProperties).accountSasToken}'
+output DFS_STORAGE_CONNECTION_STRING string = 'DefaultEndpointsProtocol=https;AccountName=${datalakeAccountName};AccountKey=${datalakeAccount.listKeys(storageApiVersion).keys[0].value};EndpointSuffix=${storageEndpointSuffix}'
 
 output DFS_SOFT_DELETE_ACCOUNT_NAME string = datalakeSoftDeleteAccountName
-output DFS_SOFT_DELETE_ACCOUNT_KEY string = listKeys(datalakeSoftDeleteAccount.id, storageApiVersion).keys[0].value
-output DFS_SOFT_DELETE_ACCOUNT_SAS string = '?${listAccountSas(datalakeSoftDeleteAccountNameTidy, storageApiVersion, accountSasProperties).accountSasToken}'
+output DFS_SOFT_DELETE_ACCOUNT_KEY string = datalakeSoftDeleteAccount.listKeys(storageApiVersion).keys[0].value
+output DFS_SOFT_DELETE_ACCOUNT_SAS string = '?${datalakeSoftDeleteAccount.listAccountSas(storageApiVersion, accountSasProperties).accountSasToken}'
 
 output FULL_ACCOUNT_NAME string = fullAccountName
-output FULL_ACCOUNT_KEY string = listKeys(fullStorageAccount.id, storageApiVersion).keys[0].value
-output FULL_ACCOUNT_SAS string = '?${listAccountSas(fullAccountNameTidy, storageApiVersion, accountSasProperties).accountSasToken}'
-output FULL_STORAGE_CONNECTION_STRING string = 'DefaultEndpointsProtocol=https;AccountName=${fullAccountName};AccountKey=${listKeys(fullStorageAccount.id, storageApiVersion).keys[0].value};EndpointSuffix=${storageEndpointSuffix}'
+output FULL_ACCOUNT_KEY string = fullStorageAccount.listKeys(storageApiVersion).keys[0].value
+output FULL_ACCOUNT_SAS string = '?${fullStorageAccount.listAccountSas(storageApiVersion, accountSasProperties).accountSasToken}'
```

Comment:
I wish we don't do this prepend of `?`, it is unrealistic in our samples. Also, hopefully, we can get rid of all these secrets soon.

Reply:
Agree. Resolving: keeping old values as this PR is fixing test resource deployment